README.md

```diff
@@ -375,10 +376,30 @@ development and testing can be done using QEMU-emulated persistent
 memory devices, see the ["QEMU and Kubernetes"](#qemu-and-kubernetes)
 section for the commands that create such a virtual test cluster.
 
-The driver does not create persistent memory Regions, but expects
-Regions to exist when the driver starts. The utility
-[ipmctl](https://github.com/intel/ipmctl) can be used to create
-Regions.
+### Persistent memory pre-provisioning
+
+The PMEM-CSI driver needs pre-provisioned regions on the NVDIMM device.
+The PMEM-CSI driver itself can not create such regions.
+The [ipmctl](https://github.com/intel/ipmctl) utility can be used to create regions,
+see the documentation of ipmctl for the details.
+With a non-interleaved creation goal, one region is created per each NVDIMM.
+In such configuration, a PMEM-CSI volume can not be larger than one NVDIMM.
+
+Example of creating non-interleaved goal using all NVDIMMs:
+```sh
+# ipmctl create -goal PersistentMemoryType=AppDirectNotInterleaved
+```
```

Comment:
Ah, so "goal" is a special term in ipmctl. I didn't know that and other readers might have the same problem. Let's write our README so that as little prior knowledge is needed as possible.

We can (and have to) use it here of course, just not in plain text.